numpy/random/mtrand/mtrand.pyx

```diff
@@ -4376,12 +4381,20 @@ cdef class RandomState:
         # not zero. We continue to use the SVD rather than Cholesky in
         # order to preserve current outputs. Note that symmetry has not
         # been checked.
+
         (u, s, v) = svd(cov)
-        neg = (np.sum(u.T * v, axis=1) < 0) & (s > 0)
+
```

Comment:
I would be the entire next part under
`if check_valid != 'ignore':`

no reason to check if we `'ignore'`